diff --git a/CMakeLists.txt b/CMakeLists.txt
index 65efe39..9901dfc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,25 +23,4 @@ configure_file(
 
 add_custom_target(uninstall
     ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake
-)
-
-########################################################################
-# Copy Files Win32 only
-########################################################################
-
-if(WIN32 AND NOT CMAKE_CROSSCOMPILING)
- if (NOT DEFINED LIBUSB_LIBRARIES)
- configure_file(
-     ${CMAKE_CURRENT_BINARY_DIR}/../libs_win32/libusb-1.0.dll
-     ${CMAKE_CURRENT_BINARY_DIR}/airspy-tools/src/libusb-1.0.dll
- COPYONLY)
- endif()
-
- if (NOT DEFINED THREADS_PTHREADS_WIN32_LIBRARY)
- configure_file(
-     ${CMAKE_CURRENT_BINARY_DIR}/../libs_win32/pthreadGC2.dll
-     ${CMAKE_CURRENT_BINARY_DIR}/airspy-tools/src/pthreadGC2.dll
- COPYONLY)
- endif()
-endif()
-
+)
\ No newline at end of file
